RELEASING.md

```diff
@@ -1,42 +1,38 @@
-  and ensure it's protected against pushing from anyone except the release
-  manager/coordinator
-    - **no PRs targeting this branch should be merged unless exceptional circumstances arise**
-- On the `RC` branch, prepare a new version section in the `CHANGELOG.md` and
-  kick off a large round of simulation testing (e.g. 400 seeds for 2k blocks)
-- If errors are found during the simulation testing, commit the fixes to `main`
-  and create a new `RC` branch (making sure to increment the `rcN`)
-- After simulation has successfully completed, create the release branch
-  (`release/vX.XX.X`) from the `RC` branch
-- Merge the release branch to `main` to incorporate the `CHANGELOG.md` updates
-- Delete the `RC` branches
-
-### Point Release Procedure
-
-At the moment, only a single major release will be supported, so all point
-releases will be based off of that release.
-
-- start on `vX.XX.X`
-- checkout a new branch `rcN/vX.X.X`
-- cherry pick the desired changes from `main`
-    - these changes should be small and NON-BREAKING (both API and state machine)
-- add entries to CHANGELOG.md and remove corresponding pending log entries
-- checkout a new branch `release/vX.X.X` based off of the previous release
-- create a PR merging `rcN/vX.X.X` into `release/vX.X.X`
-- run tests and simulations (noted in [Release Procedure](#release-procedure))
-- after tests and simulation have successfully completed, merge the `RC` branch into `release/vX.X.X`
-    - Make sure to delete the `RC` branch
-- create a PR into `main` containing ONLY the CHANGELOG.md updates
-- tag (use `git tag -a`) then push the tags (`git push --tags`)
+This document outlines the release process for https://github.com/cosmos/gaia. We use a [Long-Lived Version Branch Approach](x) because we work in parallel on a `main` branch, a `release` branch and a `release-prepare` branch. The reason for this is because our software is used to run a single live network that at any given moment only works with one version of our software. We may make a major release, but there will be a time span after that release before the software is live on the network, and we'd like our `main` branch to always be compilable and (theoretically) able to be run with the current live network (if it's in main but not a release it comes with the caveat that there may be dragons).
+
+We follow [Semver](https://semver.org/) in that any patch releases are non-breaking changes. It's important to note, that breaking changes in a Blockchain context include non-determinism. So if a code change is backwards compatible, it may still impact the amount of gas needed to execute an action, which means the change is in fact breaking as it results in a different apphash after the code is executed. It's important for non-breaking changes to be possible to be used on the live network prior to the release.
+
+
+## Long-Lived Version Branch Approach
+
+In Gaia repo, there are three categories of long-lived branches:
```

Comment:
In **Gaia** repo, there are three categories of long-lived branches:

Reply:
```suggestion
In the Gaia repo, there are three categories of long-lived branches:
```
@mmulji-ic thank you! I encourage you to use the "suggestion" feature like i'm using here for nits like this too : )